spa/src/components/settings/Integration/IntegrationCard/ActiveCampaignIntegrationCard/ConnectionModal/ConnectionModal.tsx

```diff
@@ -54,12 +60,19 @@ export function ConnectionModal({ onClose, open }: ConnectionModalProps) {
     try {
       await updateAccessTokenAndServerUrl({ accessToken, serverUrl });
       setStep('connected');
-    } catch (error) {
-      setUpdateError(
-        <>
-          <strong>Failed to save API information.</strong> Please try again.
-        </>
-      );
+    } catch (error: any) {
+      const axiosError = error as AxiosError<SaveRevenueProgramActiveCampaignUrlAndKeyValidationErrors>;
+      const message =
+        axiosError?.response?.data?.non_field_errors?.[0] === AC_URL_OR_TOKEN_VALIDATION_ERROR ? (
```

Comment:
```suggestion
        axiosError.response.data.non_field_errors?.[0] === AC_URL_OR_TOKEN_VALIDATION_ERROR ? (
```

In all cases, `axiosError` should be defined. I think also `axiosError.response.data` will always have a value?